addons/payment_adyen/models/payment_transaction.py

```diff
@@ -93,13 +93,17 @@ def _send_payment_request(self):
         if not self.provider_id.capture_manually:
             data.update(captureDelayHours=0)
 
-        # Make the payment request to Adyen
-        response_content = self.provider_id._adyen_make_request(
-            url_field_name='adyen_checkout_api_url',
-            endpoint='/payments',
-            payload=data,
-            method='POST',
-        )
+        try:
+            # Make the payment request to Adyen
+            response_content = self.provider_id._adyen_make_request(
+                url_field_name='adyen_checkout_api_url',
+                endpoint='/payments',
+                payload=data,
+                method='POST',
+            )
+        except ValidationError as e:
+            self._set_error(e)
+            return
```

Comment:
This lacks some comments. Project yourself in 6 months and re-read this code; the following questions arise and are hard to answer after just reading these three lines:
- why do we catch the error here and not in other flows?
- why do we return when the error is already caught and we could just resume with the processing?
- doesn't it pose a problem when this method is called for `online_token` payments?